ClosedXML/Excel/NamedRanges/XLNamedRange.cs

```diff
@@ -13,22 +13,33 @@ internal class XLNamedRange : IXLNamedRange
         internal XLWorkbook Workbook => _namedRanges.Workbook;
 
         public XLNamedRange(XLNamedRanges namedRanges, String rangeName, String range, String comment = null)
-            : this(namedRanges, rangeName, comment)
+            : this(namedRanges, rangeName, validateName: true, comment)
```

Comment:
`: this(namedRange, rangeName, validateName: true, range, comment)`
- this way you can leave the constructor's body empty.